src/test/java/gov/usgs/wma/waterdata/ObservationDaoIT.java

```diff
@@ -0,0 +1,141 @@
+
+	@Autowired
+	public ObservationDao observationDao;
+
+	public RequestObject request;
+	public static final String tsUniqueId = "17f83e62b06e4dc29e78d96b4426a255";
+	public TimeSeries timeSeries1 = new TimeSeries();
+	public TimeSeries timeSeries2 = new TimeSeries();
+	public TimeSeries timeSeries3 = new TimeSeries();
+
+	@BeforeEach
+	public void setup() {
+		request = new RequestObject();
+		request.setUniqueId(tsUniqueId);
+
+		String approvals = "[\"Approved\"]";
+		String grades = "[\"50\"]";
+		String qualifiers = null;
+
+		timeSeries1.setGroundwaterDailyValueIdentifier("USGS-132624144452771-17f83e62b06e4dc29e78d96b4426a255");
+		timeSeries1.setTimeSeriesUniqueId("17f83e62b06e4dc29e78d96b4426a255");
+		timeSeries1.setMonitoringLocationIdentifier("USGS-132624144452771");
+		timeSeries1.setObservedPropertyId("62610");
+		timeSeries1.setStatisticId("00001");
+		timeSeries1.setTimeStep(Date.valueOf("2008-06-03"));
+		timeSeries1.setUnitOfMeasure("ft");
+		timeSeries1.setResult("36.02");
+		timeSeries1.setApprovals(approvals);
+		timeSeries1.setQualifiers(qualifiers);
+		timeSeries1.setGrades(grades);
+
+		timeSeries2.setGroundwaterDailyValueIdentifier("USGS-132624144452771-17f83e62b06e4dc29e78d96b4426a255");
+		timeSeries2.setTimeSeriesUniqueId("17f83e62b06e4dc29e78d96b4426a255");
+		timeSeries2.setMonitoringLocationIdentifier("USGS-132624144452771");
+		timeSeries2.setObservedPropertyId("62610");
+		timeSeries2.setStatisticId("00001");
+		timeSeries2.setTimeStep(Date.valueOf("2008-06-04"));
+		timeSeries2.setUnitOfMeasure("ft");
+		timeSeries2.setResult("35.96");
+		timeSeries2.setApprovals(approvals);
+		timeSeries2.setQualifiers(qualifiers);
+		timeSeries2.setGrades(grades);
+
+		timeSeries3.setGroundwaterDailyValueIdentifier("USGS-132624144452771-17f83e62b06e4dc29e78d96b4426a255");
+		timeSeries3.setTimeSeriesUniqueId("17f83e62b06e4dc29e78d96b4426a255");
+		timeSeries3.setMonitoringLocationIdentifier("USGS-132624144452771");
+		timeSeries3.setObservedPropertyId("62610");
+		timeSeries3.setStatisticId("00001");
+		timeSeries3.setTimeStep(Date.valueOf("2008-06-05"));
+		timeSeries3.setUnitOfMeasure("ft");
+		timeSeries3.setResult("35.91");
+		timeSeries3.setApprovals(approvals);
+		timeSeries3.setQualifiers(qualifiers);
+		timeSeries3.setGrades(grades);
+	}
+
+	@Test
+	@DatabaseSetup(
+			connection="observation",
+			value="classpath:/testResult/observationDb/groundwaterDailyValue/afterInsert/")
+	@ExpectedDatabase(
+			connection="observation",
+			value="classpath:/testResult/observationDb/groundwaterDailyValue/afterDelete/",
+			assertionMode= DatabaseAssertionMode.NON_STRICT_UNORDERED)
+	public void testDelete() {
+
+		// delete existing data
+		Integer actualRowsDeletedCount = observationDao.deleteTimeSeries(request.getUniqueId());
+		assertNotNull(actualRowsDeletedCount);
+		Integer expectedRowsDeletedCount = 3;
+		assertEquals(expectedRowsDeletedCount, actualRowsDeletedCount);
+	}
+
+	@Test
+	@DatabaseSetup(
```

Comment:
Kudos on proper initialization, we should make sure this happens everywhere so the tests are not dependent on running order.

Reply:
Thanks, I took a closer look at the @DatabaseSetup annotations when I added the BaseTestDao class and made sure we are sharing setup when all tests in a class use the same source data.